recipe/meta.yaml

```diff
@@ -1,18 +1,17 @@
-{% set version = "3.5.1" %}
 
 package:
   name: r-recommended
-  version: {{ version }}
+  version: {{ r_base }}
 
 build:
   noarch: generic
   number: 1002
 
 requirements:
   host:
-      - r-base {{ version }}
+      - r-base {{ r_base }}
```

Comment:
I would completely remove the version here, as this is handled via pinning.

Reply:
good point ! hadn't considered that.